### DISCUSSION TITLE
Changing API URL

### DISCUSSION THREAD

**Discussion by metacast1** [AzuraCast/AzuraCast#5906] [Q&A]
My current API url is https://app.example.com/api/live/nowplaying/stream. 

If I change the Base Station Directory in the Administration section of the Station Profile from /var/azuracast/stations/stream to /var/azuracast/stations/radio, will the API url then change to https://app.example.com/api/live/nowplaying/radio?

Assuming that I have to restart the station, will I need to do anything else to change the API URL?

**Comment by BusterNeece** [comment]
The filesystem path and the URL are not connected like that. If you change the filesystem path, it will have no impact on the URL. Currently, the "listen" URLs are not changeable in the system.
